js/data/repository/users.js

```diff
@@ -39,10 +39,51 @@ const del = id => firestore
   .doc(id)
   .delete();
 
+const addFollowed = id => firestore.collection(collection)
```

Comment:
```suggestion
const addFollowed = id => getCurrentUser()
```
Créé cette méthode qui retourne `firestore.collection(collection).doc(fireauth.currentUser.uid)`